### ISSUE TITLE
PHP Fatal error:  Cannot make non static method PHPStan\Type\DynamicMethodReturnTypeExtension::getClass() static

### ISSUE THREAD

**Issue by simPod** [lookyman/phpstan-symfony#3]
Hey! Just installed this extension and it throws:

```
PHP Fatal error:  Cannot make non static method PHPStan\Type\DynamicMethodReturnTypeExtension::getClass() static in class Lookyman\PHPStan\Symfony\Type\ContainerInterfaceDynamicReturnTypeExtension in /path/vendor/lookyman/phpstan-symfony/src/Type/ContainerInterfaceDynamicReturnTypeExtension.php on line 18
```

Is there a bug or am I doing something wrong?

PHP71 and PHPStan 0.9

I iuncluded
`    - vendor/lookyman/phpstan-symfony/extension.neon`

And added a path to service map
```
services:
    - Lookyman\PHPStan\Symfony\ServiceMap(var/cache/dev/DevDebugProjectContainer.xml)
```

Thanks!

**Comment by lookyman** [comment]
Yep, this extension is outdated. However I plan to update it this weekend, when I should finally get some free time.

**Comment by lookyman** [comment]
Fixed by #5
